Frontend/src/locales/de.json

```diff
@@ -17,7 +17,8 @@
   },
   "labels": {
     "submit": "senden",
-    "username": "name"
+    "username": "name",
```

Comment:
Sicher, dass hier nicht auch Username stehen sollte?

Reply:
Ich war mir nicht sicher, ob Username nicht etwas zu "englisch" ist. Hab es jetzt in "Benutzername" geändert